### DISCUSSION TITLE
How can I change the dateFormat when reading xlsx stream?

### DISCUSSION THREAD

**Discussion by andreluisce** [exceljs/exceljs#2477] [Q&A]
I have the following code: 

    const workbook = new ExcelJS.Workbook();

```
    await workbook.xlsx.read(spreadsheetStream, {
        dateFormats: ['D/M/YYYY', 'D-M-YYYY', 'DD/MM/YYYY', 'DD-MM-YYYY'],
    }); 
```

In my date cell I have the date formatted as DD/MM/YYYY for example: 
`20/03/2020`

however the cell.value is `2020-01-01T00:00:00.000Z`

Is there anything that I am missing? Is it possible to have custom formats when reading XLSX?

**Comment by ghost** [comment]
When using typescript the second argument isn't even available in the type.

![image](https://user-images.githubusercontent.com/60417126/102420787-fc592d80-4056-11eb-9c86-2d453cdbb843.png)
